.gitignore

```diff
@@ -1,145 +1,145 @@
-#    `git pull`
-#    e.g., your local development database, or
-#    the S3 bucket you're using for file uploads
-#    development.
-#
-################################################
-
-config/local.js
-.env.local
-
-
-
-
-
-################################################
-# Dependencies
-#
-# When releasing a production app, you may
-# consider including your node_modules and
-# bower_components directory in your git repo,
-# but during development, its best to exclude it,
-# since different developers may be working on
-# different kernels, where dependencies would
-# need to be recompiled anyway.
-#
-# More on that here about node_modules dir:
-# http://www.futurealoof.com/posts/nodemodules-in-git.html
-# (credit Mikeal Rogers, @mikeal)
-#
-# About bower_components dir, you can see this:
-# http://addyosmani.com/blog/checking-in-front-end-dependencies/
-# (credit Addy Osmani, @addyosmani)
-#
-################################################
-
-node_modules
-bower_components
-
-
-
-
-################################################
-# Sails.js / Waterline / Grunt
-#
-# Files generated by Sails and Grunt, or related
-# tasks and adapters.
-################################################
-.tmp
-dump.rdb
-tests/e2e/screenshots
-
-
-
-
-
-################################################
-# Built artifacts
-#
-# Files generated at build-time that should not
-# be committed to source
-################################################
-assets
-
-
-
-
-################################################
-# Node.js / NPM
-#
-# Common files generated by Node, NPM, and the
-# related ecosystem.
-################################################
-lib-cov
-*.seed
-*.log
-*.out
-*.pid
-npm-debug.log
-
-
-
-
-################################################
-# Docker
-#
-# Common files generated by Docker
-################################################
-docker/data/*
-!docker/data/.gitkeep
-
-
-
-
-
-################################################
-# Miscellaneous
-#
-# Common files generated by text editors,
-# operating systems, file systems, etc.
-################################################
-
-*~
-*#
-.DS_STORE
-.netbeans
-nbproject
-.idea
-.node_history
-tests/e2e/screenshots
-tests/e2e/videos
-storybook-static
+################################################
+###############  .gitignore   ##################
+################################################
+#
+# This file is only relevant if you are using git.
+#
+# Files which match the splat patterns below will
+# be ignored by git.  This keeps random crap and
+# sensitive credentials from being uploaded to
+# your repository.  It allows you to configure your
+# app for your machine without accidentally
+# committing settings which will smash the local
+# settings of  other developers on your team.
+#
+# Some reasonable defaults are included below,
+# but, of course, you should modify/extend/prune
+# to fit your needs!
+################################################
+
+
+.husky/pre-push

```

Comment:
Why have you added this? This was intended to be in the source code. Has the prepush message been annoying? I see another PR from a different user who did the same so I'm wondering what's motivating the change

Reply:
I've had this issue a few times.  the pre-push file was annoying, so I tried to add it to the gitignore, but it didn't work.  It was a week ago, I reinstalled everything since then because I was git confused.  I didn't think this line was still in there though.  I didn't mean to keep it.
I don't know what this pre-push is and what it does...